app/services/monthly_statistics_timetable.rb

```diff
@@ -0,0 +1,69 @@
+module MonthlyStatisticsTimetable
+  # Currently, this uses the 3rd Monday of each month as agreed with Mike.
+  # It's not ideal, and may well change at a later date.
+  # It's also not cycle aware and will be needed to be updated yearly.
+
+  GENERATION_DATES = {
+    'October' => Date.new(RecruitmentCycle.previous_year, 10, 18),
+    'November' => Date.new(RecruitmentCycle.previous_year, 11, 22),
+    'December' => Date.new(RecruitmentCycle.previous_year, 12, 20),
+    'January' => Date.new(RecruitmentCycle.current_year, 1, 17),
+    'February' => Date.new(RecruitmentCycle.current_year, 2, 21),
+    'March' => Date.new(RecruitmentCycle.current_year, 3, 21),
+    'April' => Date.new(RecruitmentCycle.current_year, 4, 18),
+    'May' => Date.new(RecruitmentCycle.current_year, 5, 16),
+    'June' => Date.new(RecruitmentCycle.current_year, 6, 20),
+    'July' => Date.new(RecruitmentCycle.current_year, 7, 18),
+    'August' => Date.new(RecruitmentCycle.current_year, 8, 15),
+    'September' => Date.new(RecruitmentCycle.current_year, 9, 19),
+  }.freeze
+
+  # The date the report will be pubished is the following Thursday after
+  # the data is cut i.e. 10 days in after the generation date
```

Comment:
Confirm with Mike — he thinks maybe it should be one week later?

Reply:
Have done! It is 7 days. Have updated accordingly